src/handleIssue.ts

```diff
@@ -160,6 +161,28 @@ export const handleIssue = async (issues: Issue[], issue: Issue) => {
 		return handleIssues(issues);
 	}
 
+	if (action === Action.OpenHistory) {
+		resetDisplay();
+
+		const determinedAction = await determineHistoryIssue();
+
+		if (determinedAction === Action.Quit) {
+			return true;
+		}
+
+		const { issue, id } = determinedAction;
+
+		const hasQuit = await handleIssue(issues, issue);
+
+		if (hasQuit) {
+			return true;
+		}
+
+		removeIssueFromHistory(id);
+
+		return handleIssues(issues);
```

Comment:
This won't work for a few reasons, one of which is that `handleIssues` stores the length of the issues in a variable to be displayed in the header of the action selector. This would rewrite the length. Look into the existing implementation of undo last action to see how I did it.